introduction_to_applying_machine_learning/object2vec_document_embedding/object2vec_document_embedding.ipynb

```diff
@@ -2,13 +2,15 @@
  "cells": [
```

Comment:
<p>Do you need to keep the two <code>len()</code>  calls? If so, it would be nice to make a print statement adding what the lengths are of</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>It was a duplication, they should have the same length</p>